sectionsUtilityTool/src/main/webapp/assets/js/utils.js

```diff
@@ -30,6 +28,54 @@ var errorHandler = function (url, result) {
   return errorResponse;       
 };
 
+var getCurrentTerm = function(termData) {
+  var now = moment();
+  var currentTerm = [];
+  $.each(termData, function() {
+    //horrifying
+    //TODO: deal with Med School terms later - as this is filtering anyth8ing that does not end in '0'
+    if(moment(this.start_at).isBefore(now) && moment(this.end_at).isAfter(now)) {
+      if (this.sis_term_id !== null && this.sis_term_id !== undefined  && this.sis_term_id.slice(-1) ==='0'){
+        currentTerm.currentTermId =  this.sis_term_id;
+        currentTerm.currentTermName =  this.name;
+        currentTerm.currentTermCanvasId =  this.id;
+      }
```

Comment:
Philosophical discussion: Maybe break out of the loop here once the current term has been found so that the entire remaining list doesn't need to be processed.  For example, what if the list were very long and the one we needed was the first one?  There may be a delay while the others are evaluated.